resources/config.yml

```diff
@@ -0,0 +1,6 @@
+# Specifies weather to save item ids between runtime in order to avoid conflicts that might occur when unregistering an
+# item.
+item-id-caching: false
+# Specifies weather to save block ids between runtime in order to avoid conflicts that might occur when unregistering a
+# block.
+block-id-caching: false
```

Comment:
I don't think anyone would like to opt out of these features and rather have their items be prone to breaking if they change the registration order.